pkg/statistics/handle/autoanalyze/autoanalyze.go

```diff
@@ -516,7 +516,7 @@ func tryAutoAnalyzeTable(
 //
 // Exposed for test.
 func NeedAnalyzeTable(tbl *statistics.Table, autoAnalyzeRatio float64) (bool, string) {
-	analyzed := exec.TableAnalyzed(tbl)
```

Comment:
I guess we can add a tbl.IsAnalyzed API to simplify this code? Then we don't need to write this check everywhere.